### ISSUE TITLE
Cant run 0.3.5 from command line due to ImportError

### ISSUE THREAD

**Issue by csheppard** [Parallels/rq-dashboard#99]
```
(venv)Prodigy:web chris$ rq-dashboard -H 10.0.0.51 -D 3
Traceback (most recent call last):
  File "/Users/chris/Work/qwert/web/venv/bin/rq-dashboard", line 7, in <module>
    from rq_dashboard.cli import main
  File "/Users/chris/Work/qwert/web/venv/lib/python3.5/site-packages/rq_dashboard/__init__.py", line 2, in <module>
    import default_settings
ImportError: No module named 'default_settings'
```

**Comment by homeworkprod** [comment]
I'm seeing this as well.

#89 should fix this.

Also note that as of 0.3.5, the dashboard needs to be integrated differently into a Flask application.
Quite a breaking change for a minor version step, I'd say, especially since I can't seem to find a CHANGELOG.

**Comment by RyanBalfanz** [comment]
Also seeing this.

```
$ pip freeze | grep rq
rq==0.5.6
rq-dashboard==0.3.5
```

Does #89 fix this? Is the latest on PyPI?

**Comment by return1** [comment]
i am having the same issue, #89 fixes it.

Could you please release 0.3.6 on PyPi? Thank you!

**Comment by homeworkprod** [comment]
I second that; please release a fixed version.

**Comment by nvie** [comment]
Sorry it took this long, I just released 0.3.6 to PyPI.
